packages/file-selector/src/lib/FileSelector.tsx

```diff
@@ -135,19 +144,23 @@ export const FileSelector = ({
   children,
   customerId,
   disabled = false,
+  enableDropArea = true,
   endpoint,
   isCloud,
   label = 'Upload file',
   maxFiles,
   maxSize,
   multiple = true,
   onChange,
+  onDrop,
   onSubmit,
   onSuccess,
   onError,
   onFilePreUpload = [],
   onUploadRemove,
   retryDelays,
```

Comment:
Can also expose `metadata` that goes in `options` L176. Or maybe an options object that gets spread onto options so that individual properties don't need to be explicitly forwarded

Reply:
I added an `uploadOptions` prop for this